js/checklist.js

```diff
@@ -111,13 +113,15 @@ CheckListWidget.prototype.handleClearChecksEvent = function(event) {
 
 // When the user starts typing, change the pencil icon into a checkbox
 CheckListWidget.prototype.handleNewItemTypingEvent = function(event) {
```

Comment:
Two reasons: 1) I wanted to let you know. 2) This function is called from somewhere and I did not have the time to clean up properly … will do so.